assets/js/aplus.js

```diff
@@ -86,217 +68,240 @@ $(function() {
     $(window).bind('resize', updateMenu);
 });
 
+/**
+ * Add CustomEvent for IE 11
+ */
+(function($, window, document, undefined) {
+    "use strict";
+
+    if (typeof window.CustomEvent === "function") return false;
+
+    function CustomEvent(event, params) {
+        const bubbles = params.bubbles !== undefined ? params.bubbles : false;
+        const cancelable = params.cancelable !== undefined ? params.cancelable : false;
+        const detail = params.detail !== undefined ? params.detail : undefined;
+        const evt = document.createEvent('CustomEvent');
+        evt.initCustomEvent(event, bubbles, cancelable, detail);
+        return evt;
+    }
+    CustomEvent.prototype = window.Event.prototype;
+    window.CustomEvent = CustomEvent;
+})(jQuery, window, document);
```

Comment:
This should be the first thing on the page. The function wrap doesn't require any parameters as it's just polyfill. Not sure why it's different, but it should be from here: https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent#Polyfill

Reply:
Ah. Yeah. I wanted defaults for all parameters and not as a whole.

Reply:
I added the same format for all function wraps as some functions already had "useless" parameters that they didn't use. Should we keep the CustomEvent wrap parameterless and stick to the

Reply:
I have no idea why it's there. To my knowledge, it doesn't serve any purpose. Personally, all scopes should include only the variables they need. This documents the external interface the function depends on.

Reply:
Apparently it is used to prevent overriding of undefined. However, since we are using strict mode and it is supported in IE11, I will remove undefined from parameter list alongside with parameters that are not used in the scope.

https://stackoverflow.com/questions/33383647/javascript-undefined-as-a-function-parameter

Reply:
Actually, I think having the functions in strict mode would still not prevent defining undefined as variable outside of the function, so maybe I will just leave the functions to take undefined as parameter.

Reply:
Cool that you found the reason for it. Makes sence.. well, considering javascript is a pretty stupid language :D